src/el-data-table.vue

```diff
@@ -800,8 +804,23 @@ export default {
     searchLocatedSlotKeys() {
       return getLocatedSlotKeys(this.$slots, 'search:')
     },
+    collapseForm() {
+      return this.searchForm.reduce((res, item) => {
+        const resolveItem = {}
+
+        if (item.collapsible !== false) {
```

Comment:
```suggestion
        if (!item.collapsible) {
```
万一用户这里没用 false 而是用其他 falsy 的值呢